src/components/NavBar.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useState } from 'react';
+import BackcountryLogo from '../logos/backcountry-logo-with-text.svg';
+import NavBarSwitchingIcon from '../logos/NavBarSwitchingIcon.svg';
+import { useNavigate, useLocation } from 'react-router-dom';
```

Comment:
Library imports come at the top and then comes the file imports.
Maintain this standard across the repository

Reply:
addressed and noted